src/test/java/definitions/QuizStepDefs.java

```diff
@@ -112,4 +112,8 @@ public void iDeleteFromTheListOfQuizzes(String title) {
         getDriver().findElement(By.xpath("//ac-modal-confirmation/..//span[text()='Delete']")).click();
     }
 
+    @And("I select Show-Stopper option")
```

Comment:
This step definition should be created in your java class : ex MoiseiTextualStipperStepDefs.java